homeassistant/components/lock/lockitron.py

```diff
@@ -0,0 +1,84 @@
+import logging
+
+import requests
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.lock import LockDevice
+from homeassistant.components.lock import PLATFORM_SCHEMA
+from homeassistant.const import CONF_ACCESS_TOKEN, SERVICE_LOCK, \
```

Comment:
Make you own constante for API usage. First SERVICE means service calls and second we can change this every time and you code will not work after that.